CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing
+
+Thanks for taking the time to join our community and start contributing!
+
+The client is in early stages of development and needs more contributors. If you are interested, please read [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) first and then pick an issue from [this list](https://github.com/kubernetes-client/ruby/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22). Please comment on the issue that you are working on it. If you need help/guidance,  please check [kubernetes-client](https://kubernetes.slack.com/messages/kubernetes-client) slack channel.
+
+## Community, Support, Discussion
+
+You can reach the maintainers of this project at [SIG API Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery). If you have any problem with the package or any suggestions, please file an [issue](https://github.com/kubernetes-client/ruby/issues).
+
+### Code of Conduct
+
+Participation in the Kubernetes community is governed by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
```

Comment:
We should not be pointing directly to the CNCF's CoC

I would recommend instead adding the [standard code-of-coduct.md](https://github.com/kubernetes/kubernetes-template-project/blob/master/code-of-conduct.md) to this repo, and then link to that.  See the [kubernetes-template-project README](https://github.com/kubernetes/kubernetes-template-project/blob/master/README.md) for an example.  In the past I have audited that all repos link to the appropriate CoC by looking for that exact file.

Reply:
Updated. Ptal.